### ISSUE TITLE
NVDA's proclivity to announce the word "clickable" in Chrome is both nerve-racking and distracting, and sometimes we get it 5 times in a row

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#8451]
### Steps to reproduce:
While this can be duplicated on many websites, I'd like to provide one single example here. In Chrome, simply open https://www.pri.org/programs/it-happens , press Control+Home to move to the top of the page, press H for heading navigation, and listen as each heading gains focus.
### Actual behavior:
For many headings on this page, you'll hear the word "clickable" 5 times! For instance, "navigation landmark clickable clickable clickable clickable clickable NEWS PROGRAMS expanded  link heading  level 5 tab," navigation landmark clickable clickable clickable clickable clickable MUSIC & VARIETY PROGRAMS expanded  link heading  level 5 tab," "clickable clickable clickable clickable clickable As It Happens Logo graphic heading  level 1," "clickable clickable clickable clickable June 25: Smoke on the water link heading  level 2," etc. Reaching these headings with Up/Down also results in the announcement of the word "clickable."
### Expected behavior:
At best, NVDA should not use the word "clickable" at all or at least announce it just once. Moreover -- and preferably, NVDA shouldn't announce entering/exiting regions/landmarks when heading navigation is used. In other words, the announcement of landmarks should be preserved for quick navigation keys D and SHIFT+D, and Up/Down.
### System configuration:
Various recent Google Chrome releases, currently Chrome Version 67.0.3396.99 (Official Build) (64-bit)
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
Various versions: 2017.1 up to .4, and 2018.1 and 2018.2
#### Windows version:
Windows 7 and 10, both Pro and 64-Bit.
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Yes -- as mentioned above

**Comment by josephsl** [comment]
Hi, is this specific to Chrome, or can you reproduce this with IE and Firefox too? Also, have you considered turning off reporting of clickables in Chrome (NVDA menu/Preferences/Settings/Document formatting)? Thanks. CC @DerekRiemer

**Comment by amirsol81** [comment]
Hi. I can't duplicate this in IE 11 at all, and I don't have Firefox
for testing. However, as far as I can remember, this was also a
nonissue with Firefox in the past. Moreover, disabling the
announcement of clickables in NVDA resolves it in Chrome, but I'd like
to keep this setting enabled as I've seen its usefulness in many areas
where pressing Enter on a clickable item actually performs a task.
With this setting disabled, we can no longer know if we're on an
actually clickable item.
Thanks.

**Comment by josephsl** [comment]
Hi, understood. CC @DerekRiemer and other Chrome experts.

**Comment by amirsol81** [comment]
Thanks. If possible, please also look into limiting the announcement
of entering/exiting landmarks/regions to D and SHIFT+D. When using H
and SHIFT+H for heading navigation, it's not really necessary to hear
landmark/region names, and the fact that we're entering/exiting them.
This is really distracting, and JAWS also excludes it from heading
navigation. IMO NVDA is really chatty on webpages, especially with
Chrome.

**Comment by Brian1Gaff** [comment]
Actually there is something very strange about that page in waterfox as 
well. many headings have at least two clickables and more, If you come up 
either shift h then some that seem only to have a couple suddenly get 
several more.
 However if you go around the page with cursors you hear none at all.
 Its as if there is hidden content.
 Most odd.
Brian

**Comment by ehollig** [comment]
There is discussion about improving the announcement of clickable in #7430. I am not sure if I should close this issue as a duplicate or not

**Comment by amirsol81** [comment]
Hello, the problem is that the so-called clickable issue persists
despite the lengthy discussion at 7430. So my suggestion is that this
be kept open. I can provide numerous other examples for this if
desired.

**Comment by amirsol81** [comment]
As another example, if, in Chrome, you perform a search on Google, NVDA will read the first heading after the "main landmark Search Results heading  level 2" preceded by the word "clickable" twice upon pressing H for heading navigation. All other headings after that are read preceded by the word "clickable" once. The exception to this is if subsequent search results, presented as separate headings, are from one website. Put simply, just see how NVDA reads the heading after the "main landmark Search Results heading  level 2" upon pressing H.

**Comment by leonardder** [comment]
As the initial problem raised in #7430 is identical to the issue here, I'agree with @ehollig m and am closing this as a duplicate.
